sources/packages/backend/apps/queue-consumers/src/processors/virus-scan/_tests_/virus-scan.processor.e2e-spec.ts

```diff
@@ -40,12 +46,147 @@ describe(describeProcessorRootTest(QueueNames.FileVirusScanProcessor), () => {
     jest.resetAllMocks();
   });
 
+  it("Should throw an error when the student file is not found during virus scanning.", async () => {
+    // Arrange
+    const studentFile = createFakeStudentFileUpload();
+    studentFile.virusScanStatus = VirusScanStatus.InProgress;
+    await db.studentFile.save(studentFile);
+    clamAVServiceMock.scanFile = jest.fn(() =>
+      Promise.resolve({
+        isInfected: null,
```

Comment:
Do we need this? Same for the next 3 tests (everywhere where an error is thrown).

Reply:
Updated.